manageiq-smartstate.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "azure-armrest",      "~> 0.9.6"
+  spec.add_dependency "azure-armrest",      "~> 2.1.0"
```

Comment:
I think that's the rest-client version ;)
Can you update azure-armrest to just `0.10` (aka not 0.10.0)

Reply:
yeah sorry got it